lib/katello/actions/actions.rb

```diff
@@ -11,6 +11,7 @@ module Actions
 
     require 'katello/actions/repository_create'
     require 'katello/actions/repository_destroy'
+    require 'katello/actions/repository_export'
     require 'katello/actions/repository_sync'
   end
 end
```

Comment:
i don't think this file is actually used any more, good job finding it ;)  mind deleting it?